### PR TITLE
empty value in quotes is not an error

### PR DESCRIPTION
**Pull request by cv711**

### CODE REVIEW

File: authres_test.go [hunk 1]
```diff
@@ -44,6 +44,19 @@ func TestParseAuthenticationResults(t *testing.T) {
 				},
 			},
 		},
+		{
+			`example.com; spf=neutral smtp.mailfrom=""`,
```
Comment (narqo) [inline authres_test.go:48]: Don't you want to add a test case where an empty value was passed without quotes? It must fail, doesn't it?

Comment (cv711) [reply authres_test.go:48]: yes it does, but I was planning to do it after the next pull request where we have some mechanism of collecting errors and then compare against that.
